### DISCUSSION TITLE
Implement pop-up pins

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#148] [Ideas]
Hi Magne,
A last, low priority suggestion for today.
Think about adding pins like in GoogleEarth, with text, media and external links. This could help adding info like pictures, videos, links to websites, implementing the information offered by the tactical maps themselves.
Regards
Federico

**Comment by kjellmf** [comment]
This is already on my todo-list. Added #150 to keep track of it.
